src/components/box/Box.tsx

```diff
@@ -65,17 +65,14 @@ export function Box({
           return;
```

Comment:
If mode is draw we should also ignore the event and let it propagate.

Otherwise we can't draw over editable ROI.


https://github.com/zakodium-oss/react-roi/assets/4118690/977acfa6-f24e-4416-a3f8-8e57c43e11d0